src/process_model_output.R

```diff
@@ -1,20 +1,20 @@
+# Code for total storage daily build
+
 library(ncdf4)
 library(dplyr)
 
 args <- commandArgs(trailingOnly=TRUE)
-today <- args[1]
-
-#### Code for total storage daily build
-#This section is code for what I think will replace the precip code when the percentile code is complete.
-# test for now b/c of the data we have
-#today <- "2019-07-31"
+# today <- args[1] #today <- "2019-07-31"
```

Comment:
Does this need to be uncommented?

Reply:
And/or converted to a 'days since' value?

Reply:
oh my it does! Does not need to be "days since" though